helx/charts/roger/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
```

Comment:
Is this a temp file?

Reply:
This is lets rebuild the charts/ directory to the state specified in the lock file (https://helm.sh/docs/helm/helm_dependency_build/). This was generated using helm dependency update (https://helm.sh/docs/helm/helm_dependency_update/)  , I think the advantage is more precision on the dependency version (?)

Reply:
I believe this file will be used to download the tar archives of the dependencies.  I'll need to do a "helm dependency update" when I add this to the devops helm repo.